README.md

```diff
@@ -21,7 +21,7 @@ Node.js client library to use the Watson APIs.
   * [Examples](#examples)
   * [IBM Watson Services](#ibm-watson-services)
     * [Authorization](#authorization)
-    * [Assistant]
+    * [Assistant] (#assistant)
```

Comment:
The space in between the [] and () needs to be removed

Reply:
Good catch